### ISSUE TITLE
[META 355] Support Prometheus Metric Libraries

### ISSUE THREAD

**Issue by AlexanderWert** [elastic/apm-agent-go#866]
Implement https://github.com/elastic/apm/issues/355

**Comment by matschaffer** [comment]
I'm fairly certain this works already. @henrikno helped set it up for a couple of our go services.

**Comment by axw** [comment]
Yup, IIRC the only missing piece is support for histograms, which we can add after support lands in APM Server in 7.14.

We should probably also change the handling of summary metrics at some point too. Currently they're send as multiple metrics, but probably should be recorded in a single [`aggregate_metric_double`](https://www.elastic.co/guide/en/elasticsearch/reference/current/aggregate-metric-double.html) field.
